client/android/app/build.gradle

```diff
@@ -84,7 +84,7 @@ def enableProguardInReleaseBuilds = false
 
 android {
     compileSdkVersion 23
-    buildToolsVersion "23.0.1"
+    buildToolsVersion '25.0.0'
```

Comment:
Are you talking about "to take all of the latest features we recommend you upgrade" prompt?
 I'd prefer to just ignore that and stick with the version React Native supports, lest we start running into build issues. You can see some discussion around this here: https://github.com/facebook/react-native/pull/12930.